irohad/synchronizer/impl/synchronizer_impl.cpp

```diff
@@ -45,12 +51,13 @@ namespace iroha {
           if (blocks.empty()) {
             log_->info("Downloaded an empty chain");
             continue;
+          } else {
+            log_->info("Successfully downloaded {} blocks:", blocks.size());
```

Comment:
I thing, this log entry should be moved under `if` below - we are not very interested in the fact of block download, only in successful one, and here we are not sure they will be applied

Reply:
let it be as is)